dhis-2/dhis-services/dhis-service-dxf2/pom.xml

```diff
@@ -202,8 +202,13 @@
       <artifactId>javax.annotation-api</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+       <groupId>org.apache.logging.log4j</groupId>
+       <artifactId>log4j-slf4j-impl</artifactId>
+      <scope>provided</scope>
```

Comment:
if we define the scopes in the root pom, we would not need to repeat that in each pom AFAIK

Reply:
Done